osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -249,7 +250,28 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
             if (userTriggered || Time.Current < HitObject.EndTime)
                 return;
 
-            ApplyResult(r => r.Type = NestedHitObjects.Any(h => h.Result.IsHit) ? r.Judgement.MaxResult : r.Judgement.MinResult);
+            if (HitObject.IgnoreJudgement)
```

Comment:
Hmm, going to have to have a think about this one. I'm not sure about polluting the `DrawableSlider` class with this kind of stable knowledge... Wonder how well this would work as a function we can assign from the mod itself?

If this is to remain here, I'd amend the comment just below to mention that the non-"ignore" case is the legacy implementation, because the variable naming feels a bit inverse otherwise (see my mistaken comment from before this review).

Reply:
I've kept it as is and adjusted the comments. I'm not sure how far to go in that case because `DrawableSliderHead` has a similar property added for follow circle tracking.